shell/egl_window.cc

```diff
@@ -107,9 +107,9 @@ EglWindow::EglWindow(size_t index,
   m_fps_idx = 0;
   m_fps_counter = 0;
 
-  //  this->m_callback = wl_surface_frame(this->m_surface);
-  //  wl_callback_add_listener(this->m_callback, &frame_listener, this);
-  wl_surface_commit(this->m_surface);
+  m_callback = wl_surface_frame(this->m_surface);
+  wl_callback_add_listener(m_callback, &frame_listener, this);
+  wl_surface_commit(m_surface);
```

Comment:
Probably this wouldn't need any comments from the previous patch. Also wouldn't `this->m_surface == m_surface`?

Reply:
yes, left over from prior patch.  remove